pkg/droplets/droplets.go

```diff
@@ -111,3 +112,28 @@ func (svc *dropletSvc) list(all otto.FunctionCall) otto.Value {
 	}
 	return v
 }
+
+func (svc *dropletSvc) createMultiple(all otto.FunctionCall) otto.Value {
+	vm := all.Otto
+
+	arg := all.Argument(0)
+
+	req := godojs.ArgDropletMultiCreateRequest(vm, arg)
+
+	droplets, err := svc.svc.CreateMultiple(svc.ctx, req.Names, req.Region, req.Size, req.Image.Slug, droplets.UseGodoMultiCreate(req))
+	if err != nil {
+		ottoutil.Throw(vm, err.Error())
+	}
+
+	var d = make([]otto.Value, 0)
+	for _, droplet := range droplets {
+		d = append(d, godojs.DropletToVM(vm, droplet.Struct()))
+	}
+
+	v, err := vm.ToValue(d)
+	if err != nil {
+		ottoutil.Throw(vm, err.Error())
```

Comment:
What's going to happen to the created droplets here?  Do we need to clean up?

Reply:
@table-delete Not in this project. In canary, they'll get automagically deleted by the garbage collector.